Tools/Scripts/generate-dirty-files

```diff
@@ -153,14 +173,14 @@ def find_all_results(args):
         except subprocess.CalledProcessError as e:
             sys.stderr.write(f'{e.output}')
             sys.stderr.write(f'Could not find results for {project}\n')
-            return -1
+            break
```

Comment:
Do we really want to upload nothing in the case where we fail to get results all?

Reply:
Ah I should probably be using `continue` here.